### ISSUE TITLE
Make force-pushes to a PR'd branch cancel the previous CI job(s)

### ISSUE THREAD

**Issue by richardpringle** [ava-labs/firewood#261]
It makes sense to let a job finish when someone pushes new commits to an open PR so that you can see if CI passed on a previous commit, but it doesn't make sense to do the same when someone force-pushes. In the case where someone force-pushes, the previous job for that branch should be cancelled. 

I'm not sure how easy it is to do this, but we should at least explore the options.

**Comment by rkuris** [comment]
I think it already does this!

**Comment by richardpringle** [comment]
> I think it already does this!

I don't think so haha. I will verify next time I force-push and get some screenshot evidence

**Comment by rkuris** [comment]
We force push very infrequently and so I'm closing this as wontfix.
